ansible/roles/jenkins-pipeline/templates/create_credentials_config.xml.j2

```diff
@@ -28,7 +28,7 @@
         <hudson.model.StringParameterDefinition>
           <name>JENKINS_SLAVE_USER_NAME</name>
           <description>User name used to access Jenkins slaves.</description>
-          <defaultValue>jenkins</defaultValue>
+          <defaultValue>{{user_name}}</defaultValue>
```

Comment:
Those are the credentials used to access the slaves, and so should contain the user name of those nodes, `{{jenkins_slave_user_name}}`